packages/preview-service/observability/logging.js

```diff
@@ -0,0 +1,32 @@
+// Note logging is imported by www & ts-www, prior to init() being called
+// so we can't use imports with '@' etc., as they aren't yet defined.
+const pino = require('pino')
+
+const Logger = pino({
```

Comment:
Another kinda minor thing that does still bother me 😅 PascalCase is commonly used to refer to types (classes, Vue components), while in this case Logger is not a type/class, but an instance of one - a logger instance, and for those the standard variable naming rules of camelCase apply.

Especially if other (child) loggers are camelCased, I think all of the PascalCased `Logger`s should be renamed to `logger` as well